src/box/iproto.cc

```diff
@@ -2977,6 +3021,8 @@ iproto_stats_get(struct iproto_stats *stats)
 		cfg_msg.stats = &thread_stats;
 		iproto_do_cfg(&iproto_threads[i], &cfg_msg);
 		iproto_stats_collect_thread_stats(stats, &thread_stats);
+		stats->requests_in_progress +=
+			iproto_threads[i].tx.requests_in_progress;
```

Comment:
Can you move it to iproto_stats_collect_thread_stats (which I asked you to rename to iproto_stats_add) so that all stats are accumulated in the same way?

Reply:
Fixed